src/themis/secure_comparator.c

```diff
@@ -38,7 +38,7 @@ static themis_status_t ed_sign(uint8_t pos, const uint8_t *scalar, uint8_t *sign
 {
 	uint8_t r[ED25519_GE_LENGTH];
 	ge_p3 R;
-	uint8_t k[64];
+	uint8_t k[64]; //temp buffer for group elements and hash
```

Comment:
it's annoyed to read code with such variable names for people who never saw another crypto libraries (like me). On my opinion, if we write code for other crypto developers than it maybe good way but otherwise I vote for good understandable variable names